dipy/reconst/dki.py

```diff
@@ -359,6 +361,75 @@ def _F2m(a, b, c):
     return F2
 
 
+def _directional_kurtosis(dt, MD, kt, V, min_diffusivity=0, min_kurtosis=-1):
+    r""" Helper function that calculate the apparent kurtosis coefficient (AKC)
+    in each direction of a sphere for a single voxel.
+
+    Parameters
+    ----------
+    dt : array (6,)
+        elements of the diffusion tensor of the voxel.
+    MD : float
+        mean diffusivity of the voxel
+    kt : array (15,)
+        elements of the kurtosis tensor of the voxel.
+    V : array (g, 3)
+        g directions of a Sphere in Cartesian coordinates
+    min_diffusivity : float (optional)
+        Because negative eigenvalues are not physical and small eigenvalues
+        cause quite a lot of noise in diffusion based metrics, diffusivity
+        values smaller than `min_diffusivity` are replaced with
+        `min_diffusivity`. defaut = 0
+    min_kurtosis : float (optional)
+        Because high amplitude negative values of kurtosis are not physicaly
+        and biologicaly pluasible, and these causes huge artefacts in kurtosis
```

Comment:
Typo in "plausible". 

"causes" => "cause". 

"high amplitude" => "high-amplitude"

I would also drop the word "huge" (https://twitter.com/search?q=%23yuuge :-D)